library/Ivoz/Provider/Infrastructure/Redis/Jobs/Invoicer.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Ivoz\Provider\Infrastructure\Redis\Jobs;
+
+use Ivoz\Core\Infrastructure\Persistence\Redis\RedisMasterFactory;
+use Ivoz\Provider\Domain\Job\InvoicerJobInterface;
+use Psr\Log\LoggerInterface;
+
+class Invoicer implements InvoicerJobInterface
+{
+    private $redisMasterFactory;
+    private $redisDb;
+    private $logger;
+    private $id;
+
+    public function __construct(
+        RedisMasterFactory $redisMasterFactory,
+        int $redisDb,
+        LoggerInterface $logger
+    ) {
+        $this->redisMasterFactory = $redisMasterFactory;
+        $this->redisDb = $redisDb;
+        $this->logger = $logger;
+    }
+
+    public function setId(int|string $id)
```

Comment:
I think this sets the Invoice id that should be int in all cases. Why is string allowed here?